From a9e16e6d830369572f90d758bb596de6fd8f616b Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 22 Aug 2015 19:09:56 -0700
Subject: [PATCH] Fix link error on OSX

Error was: "error: declaration of 'tolower' has a different language linkage"
---
 src-fceux/utils/xstring.cpp | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/src-fceux/utils/xstring.cpp b/src-fceux/utils/xstring.cpp
index 57ffc0e..8f1607d 100644
--- a/src-fceux/utils/xstring.cpp
+++ b/src-fceux/utils/xstring.cpp
@@ -23,16 +23,6 @@
 
 #include "xstring.h"
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-int tolower(int);
-
-#ifdef __cplusplus
-}
-#endif
-
 //this code was taken from WINE (LGPL)
 //http://google.com/codesearch?hl=en&q=+lang:c+splitpath+show:CPvw9Z-euls:_RSotQzmLeU:KGzljMEYFbY&sa=N&cd=9&ct=rc&cs_p=http://gentoo.osuosl.org/distfiles/Wine-20050524.tar.gz&cs_f=wine-20050524/programs/winefile/splitpath.c
 void splitpath(const char* path, char* drv, char* dir, char* name, char* ext)
-- 
1.9.5 (Apple Git-50.3)

